### ISSUE TITLE
Consortium IDs for clients are serialized as upper case, resulting in search and display errors

### ISSUE THREAD

**Issue by codycooperross** [datacite/lupo#871]
### Describe the bug

A behavior change is causing consortium IDs to be serialized as uppercase in client serializations. This causes the Repositories screen in Fabrica to omit repositories for Consortium Organizations.

#### Expected Behaviour

Former, expected API response:

```
"consortium": {
"data": {
"id": "nasaco",
"type": "providers"
}
},
```

#### Current Behaviour

API response for client updated recently:

```
"consortium": {
"data": {
"id": "NASACO",
"type": "providers"
}
},
```

#### Steps to Reproduce

See this API call, performed in bracco: https://api.datacite.org/repositories?certificate=&client-type=&consortium-id=nasaco&page%5Bnumber%5D=1&page%5Bsize%5D=25&provider-id=&query=&repository-type=&size=25&software=&sort=&year=

NASAARMD.AFPDA is not included in the response because the consortium ID contains uppercase characters: https://api.datacite.org/clients/NASAARMD.AFPDA

### Context (Environment)

This mainly appears to impact Consortium accounts in Fabrica.

#### Hypothesis

This may impact other serializations, like prefixes. 

#### Possible Implementation

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2zpcl)

**Comment by svogt0511** [comment]
Suggestion:  The fix for case insensitive searching will help, #1002, in addition to addition to PR #993 (Always return lowercase provider id in provider relationship data).  We might want a similar fix other serializers to return lowercase ids (consortium id, repository id),

**Comment by codycooperross** [comment]
Resolved by datacite/datacite#1003
